js/ez-browse.js

```diff
@@ -0,0 +1,66 @@
+(function () {
+    /**
+     * <ez-browse> represents a clickable element in the application which will fire
+     * the `ez-contentDiscover` event on click. This event is responsible for launching
+     * the universal discovery widget component. Some config and listeners are given to the event
```

Comment:
*Some config and listeners* ? no, that's very precise actually, please define what's happening